frontend/src/components/misc/PipelinesYamlEditor.tsx

```diff
@@ -71,60 +72,29 @@ export const monacoYamlOptions = {
       },
 
       // And the URI will be linked to as the source.
-      uri: 'http://example.com/schema-name.json', // '../../benthos-schema.json',
+      uri: 'http://example.com/redpanda-connect-schema.json', // '../../benthos-schema.json',
```

Comment:
shouldn't you add an actual url like the benthos-schema?

Reply:
which one?